iexec-worker-standalone-library/src/main/java/com/iexec/standalone/api/SchedulerClient.java

```diff
@@ -16,12 +16,12 @@
 
 package com.iexec.standalone.api;
 
+import com.iexec.common.config.PublicConfiguration;
```

Comment:
This is a reversion from PR 699 and should not happen. Why do you need to go back to this import ?